presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -111,6 +111,18 @@ public void testDecimalPredicatePushdown()
         }
     }
 
+    @Test
+    public void testInsertVarbinary()
+            throws Exception
+    {
+        try (AutoCloseable ignoreTable = withTable("test_varbinary",
+                "(x varbinary(max))")) {
+            assertUpdate("INSERT INTO test_varbinary VALUES (X'65683F'), (NULL)", 2);
+            assertQuery("SELECT * FROM test_varbinary",
```

Comment:
Can this be tested with `DataTypeTest`? Maybe we can have `TestSqlServerTypeMapping` stub?
(see https://github.com/prestosql/presto/issues/565)

Reply:
@findepi I avoided `TestSqlServerTypeMapping` considering #4734, but moved the test to the class.

Reply:
Right. Let's have a TODO in the code, to move the method over there, while `TestSqlServerTypeMapping` exists

Reply:
Oh, i see you called the test class `TestSqlServerTypeMappi`. Fine.